test/OrchardCore.Commerce.Tests.UI/UITestBase.cs

```diff
@@ -19,19 +19,24 @@ protected UITestBase(ITestOutputHelper testOutputHelper)
         ExecuteTestAsync(testAsync, browser, SetupHelpers.RunSetupAsync, changeConfigurationAsync);
 
     protected override Task ExecuteTestAsync(
-        Action<UITestContext> test,
+        Func<UITestContext, Task> testAsync,
         Browser browser,
-        Func<UITestContext, Task<Uri>>? setupOperation = null,
-        Action<OrchardCoreUITestExecutorConfiguration>? changeConfiguration = null) =>
+        Func<UITestContext, Task<Uri>> setupOperation,
+        Func<OrchardCoreUITestExecutorConfiguration, Task> changeConfigurationAsync) =>
         base.ExecuteTestAsync(
-            test,
+            testAsync,
             browser,
             setupOperation,
-            configuration =>
+            async configuration =>
             {
                 configuration.AccessibilityCheckingConfiguration.RunAccessibilityCheckingAssertionOnAllPageChanges = true;
+                configuration.AccessibilityCheckingConfiguration.AxeBuilderConfigurator += axeBuilder =>
+                    AccessibilityCheckingConfiguration
+                        .ConfigureWcag21aa(axeBuilder)
+                        .DisableRules("color-contrast", "html-has-lang");
+
                 configuration.UseSqlServer = true;
```

Comment:
@Psichorex Do you remember why you made this use SQL Server instead of simply SQLite?

Reply:
As far as I can see this line is not necessary. (at least all the UI tests worked for me locally without it) Could you remove it? And if that works, please update the workflow config [here](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/8990a6ec07653d81c1c7bdd21d3b0a662b876483/.github/workflows/build-and-test.yml#L17) so SQL Server is not installed needlessly during the CI runs.

Reply:
I think it's partially necessary because of the setup page:
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/51411356/36a636af-66d7-4a04-9669-e8d9892dc22d)

Though I could remove the second rule from it without issues it seems.

Reply:
What does that have to do with the `configuration.UseSqlServer = true;` line?

Reply:
I may have forgotten how PR comments on GitHub work for a second. Nevermind.